EIPS/eip-3475.md

```diff
@@ -0,0 +1,449 @@
+---
+eip: 3475
+title: Multiple Callable Bonds
+author: Yohji Sakamoto (@sgmfinance)
+discussions-to: https://github.com/ethereum/EIPs/issues/3467
+type: Standards Track
+category: ERC
+status: Draft
+created: 2021-04-05
+---
+
+## Simple Summary
+
+A standard interface for contract, that manage multiple callable bonds. A single contract includes any given number of bond classes, bond nonce, bond balance of an address. This standard provides independent functions to read, transfer any collection of bonds, as well as allow bonds to be redeemed from the bond issuer if certain conditions are met. This token standard can replace current ERC-20 LP token. ERC-659 has more complex data structure, which will allow the LP token to store more information, and allow the developer to build more sophisticated logic for the redemption and reward system of the defi project in question.
```

Comment:
While this isn't a blocker for draft stage, please shorten this to one sentence in the future.